### ISSUE TITLE
Only head position and hydra for controls

### ISSUE THREAD

**Issue by daurocg** [cboulay/PSMoveService#480]
Hi all

firt, great job with this

second, i want use my razer hydra for the controlrs and ps move for head position with 2 cameras, i have to use PSMoveFreepieBridge, but i have to do all configuration with 3 ps move?

thanks
regards

**Comment by iamapie13** [comment]
no, you don't even need a PS Move controller, you can use a ping pong ball and make a virtual HMD
